src/Escaper.php

```diff
@@ -30,7 +30,7 @@
  *
  * @final
  */
-class Escaper
+class Escaper implements EscaperInterface
```

Comment:
I'd like to see interface methods have the docblocks replaced with `/** @inheritDoc */` so that no one is tempted to update docs on the implementation.

The `MissingOverrideAttribute` Psalm issue should be added to `psalm.xml` as a globally suppressed issue. Example: https://github.com/laminas/laminas-filter/commit/e7c4512119daaff3f869d883d55976ceac70165a